### PR TITLE
Cleanup recipe

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes https://github.com/conda-forge/torchvision-feedstock/pull/36

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I think the build string is a little wrong. or at least inconsistent with what we do at pytorch. I want to reconciliate before merging.

**Comment by hmaarrfk** [comment]
@conda-forge/torchvision how do people feel about the change to the build number?

I feel like we should keep it in sync with pytorch's convention, but with the run export, I'm worried that it might throw some recipes out of wack.

If the problem is only a few recipes, then I don't think it is a big deal, they will eventually get migrated over, but I wanted to hear your thoughts.

**Comment by hmaarrfk** [comment]
maybe it is best to wait until the next version of pytorch for this cleanup, we will have a migrator to help us catch the cleanups in downstream recipes.

the other question I have is why we even have this run exports in the first place. it seems a little tricky to get right

**Comment by hmaarrfk** [comment]
I also removed the run_exports from the latest recipe, seems rather dangerous, and we ask people to do the doulbe pytorch pinning anyway.

**Comment by h-vetinari** [comment]
Can we keep the `outputs:`? If we ever add different outputs (e.g. for the C++ lib, or with/without mpeg support, cf. #22), then we'd could avoid a whole lot of churn...

**Comment by hmaarrfk** [comment]
> Can we keep the `outputs:`? If we ever add different outputs (e.g. for the C++ lib, or with/without mpeg support, cf. #22), then we'd could avoid a whole lot of churn...

ok

**Comment by hmaarrfk** [comment]
@h-vetinari let me know if you have any other concerns.

**Review by h-vetinari — APPROVED** [review]
LGTM, thanks!

**Comment by h-vetinari** [comment]
On second thought, a nit: please squash the commits in such a way that we avoid the churn I mentioned of going back-and-forth between multiple resp. single output in the commit history.

**Comment by hmaarrfk** [comment]
done. thanks

File: recipe/meta.yaml
Comment (h-vetinari) [inline recipe/meta.yaml:69]: Presumably this is handled by a run-export? Any other reason for removing it?

File: recipe/meta.yaml
Comment (h-vetinari) [inline recipe/meta.yaml:104]: nit: stric -> strict

**Review by h-vetinari — APPROVED** [review]
One more question and a really irrelevant nit. Affirming LGTM in any case, thanks!

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml:69]: run exports it is.

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml]: Fixed.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please restart cis

**Comment by ngam** [comment]
@hmaarrfk can this potentially close #40? I see that we got rid of all pinning in this one --- any idea how they would sync going forward?

File: recipe/meta.yaml
Comment (ngam) [inline recipe/meta.yaml:41]: Why both?

File: recipe/meta.yaml
Comment (ngam) [reply recipe/meta.yaml:69]: Can you elaborate how this was handled because I am not following the logic...

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml:41]: so that it picks up the cuda/cpu difference.

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml:69]: https://conda-forge.org/docs/maintainer/pinning_deps.html?highlight=run_exports#specifying-run-exports

Comment (ngam) [reply recipe/meta.yaml:69]: Yes, but I don't see run_exports in the meta... I thought it would be there

Comment (hmaarrfk) [reply recipe/meta.yaml:69]: https://github.com/conda-forge/ffmpeg-feedstock/blob/master/recipe/meta.yaml#L21

**Pull request merged**
